numpy/core/tests/test_function_base.py

```diff
@@ -402,3 +402,8 @@ def test_object(self):
         stop = array(2, dtype='O')
         y = linspace(start, stop, 3)
         assert_array_equal(y, array([1., 1.5, 2.]))
+
+    def test_round_negative(self):
+        y = linspace(-1, 3, num=10, endpoint=False, dtype=int)
+        t = array([-1, -1, -1, 0, 0, 1, 1, 2, 2, 3], dtype=int)
```

Comment:
Hmm, seems we have another bug here - the endpoint is included!

Reply:
Maybe set `num=8` so that this test makes sense, and open a new issue about `endpoint`?

Reply:
Oh, I'm sorry. I wrote the wrong test code. Endpoint error does not exist.